src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractFileSetCheck.java

```diff
@@ -174,6 +174,6 @@ protected final void fireErrors(String fileName) {
         final SortedSet<LocalizedMessage> errors = messageCollector
                 .getMessages();
         messageCollector.reset();
-        getMessageDispatcher().fireErrors(fileName, errors);
+        getMessageDispatcher().fireErrors(fileName, new FileProcessingResult(null, errors));
```

Comment:
as we need to pass null as the constructor parameter it means that this is a problem in the design. Please, create one more constructor which receives one parameter and sets fileText to null. The constructor can be reused in the FileProcessingResult itself.